docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md

```diff
@@ -0,0 +1,37 @@
+# Python Syntax Changelog
+
+**Purpose:** This document collates specific language changes from Python 3.8 up. It is intended to
+aid in the investigation of why Python is functional in 3.7 and earlier, but not later versions.
+
+**This document is not complete.**
+
+## Python 3.0
+
+Python 3.0 was intentionally developed to be backwards compatible. At the establishment of the
+Python project team, we were informed that versions prior to 3.8 functioned as intended with the
+translator. This leads to the thinking that the Python 3.8 update is specifically responsible for
+the failure in compatibility.
+
+## Python 3.8
+
+**Assignment expression** introduces new syntax, used by :=, to assign values to variables as part
+of a larger expression.
+
+if (n := len(a)) \&gt; 10: print(f&quot;List is too long ({n} elements, expected \&lt;= 10)&quot;)
```

Comment:
**suggestion:** Should this be wrapped in a code [block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)? 🤔